packages/generated-react/buf.gen.yaml

```diff
@@ -25,5 +25,6 @@ plugins:
     path: ../protoc-gen-connect-query-react/bin/protoc-gen-connect-query-react
     opt:
       - target=ts
+      - import_extension=none
```

Comment:
Any specific reason we changed this? It's not a big deal, but i'd like to make sure we're only making the necessary changes.

Reply:
And since no other plugins have this enabled, i'd rather we be consistent.

Reply:
> And since no other plugins have this enabled, i'd rather we be consistent.

Ah let me change it back

Reply:
I just updated it, thank you @paul-sachs